.github/workflows/upload-tests-manifest.yml

```diff
@@ -6,12 +6,7 @@ name: Upload bundler test manifests to areweturboyet.com
 on:
   schedule:
     - cron: '0 8 * * *'
-  workflow_dispatch:
```

Comment:
were these unused?

Reply:
We don't need to specify any options here, but we do need to configure this so that we can more easily test it by manually scheduling it.

> To run a workflow manually, the workflow must be configured to run on the workflow_dispatch event.

https://docs.github.com/en/actions/managing-workflow-runs-and-deployments/managing-workflow-runs/manually-running-a-workflow#configuring-a-workflow-to-run-manually